tests/integration/test.basics.js

```diff
@@ -1106,5 +1106,21 @@ adapters.forEach(function (adapter) {
         });
       });
     }
+
+    if (typeof process !== 'undefined' && !process.browser) {
```

Comment:
Is there a reason this test only runs in Node? Ideally `PouchDB.plugin()` would throw in the browser too.

Also it may make it easier to test this; you can do `npm run dev` and then open `http://localhost:8000/tests/integration?grep=unique_word_in_my_test_name` and then test in your preferred browser.